routes/api/index.js

```diff
@@ -386,18 +396,67 @@ router.get('/getRegistrationData/:eventID', checkIfAuthenticated, async (req, re
             res.status(200).send({
                 status: 'success',
                 data: {
-                    registered: !!registration_data
+                    registered: !!registration_data,
+                    extra_data: registration_data?.extra_data
                 }
             });
-        } else {
-            const team_data = await Team.findById(registration_data?.team_id).exec();
+        } else if (registration_data) {
+            const team_data = await Team.findById(registration_data.team_id).exec();
+            // const member_ids = (await EventRegistration.find({team_id: registration_data.team_id}).select('participant_id').exec())
+            //     .map(doc => doc.participant_id);
+            // const member_names = (await User.find({ _id: { $in: member_ids } }).select('name').exec())
+            //     .map(doc => doc.name);
+            const member_info = await EventRegistration.aggregate([
+                {
+                  '$match': {
+                    'team_id': '6f3d4f'
+                  }
+                }, {
+                  '$lookup': {
+                    'from': 'users', 
+                    'localField': 'participant_id', 
+                    'foreignField': '_id', 
+                    'as': 'data'
+                  }
+                }, {
+                  '$group': {
+                    '_id': '$team_id', 
+                    'members': {
+                      '$push': {
+                        '$arrayElemAt': [
+                          '$data', 0
+                        ]
+                      }
+                    }
+                  }
+                }, {
+                  '$project': {
+                    'members': {
+                      '$map': {
+                        'input': '$members', 
+                        'as': 'member', 
+                        'in': '$$member.name'
+                      }
+                    }
+                  }
+                }
+              ]).exec();
             res.status(200).send({
                 status: 'success',
-                data: !!registration_data ? {
+                data: {
                     team_id: team_data._id,
                     team_name: team_data.name,
-                    meeting_link: team_data.meeting_link
-                } : { registered: false }
+                    meeting_link: team_data.meeting_link,
```

Comment:
meeting link should be inside team_extra_data

Reply:
yes